jreleaser.yml

```diff
@@ -66,6 +67,7 @@ files:
 
 distributions:
   jbang:
+    executableExtension: cmd
```

Comment:
what does this mean?

Reply:
By default JReleaser uses .bat as the file extension for executables/launchers on Windows. JBang uses .cmd. The leading . is not necessary in the config